src/bootstrap/test.rs

```diff
@@ -632,6 +632,27 @@ impl Step for Clippy {
 
         cargo.add_rustc_lib_path(builder, compiler);
 
+        if builder.try_run(&mut cargo.into()) {
+            // The tests succeeded; nothing to do.
+            return;
+        }
+
+        if !builder.config.cmd.bless() {
+            std::process::exit(1);
+        }
+
+        println!("running `cargo dev bless` on clippy tests");
```

Comment:
Is there a reason for an explicit message here? compiletest blessing is silent -- is there a reason we can't pull that off here?

FWIW I find it a bit interesting that unlike compiletest this is a separate command etc -- maybe @Manishearth or someone familiar with clippy can say why? I'd expect this to be just a flag passed in to compile test when running clippy's tests.

Reply:
No particular reason, I'll remove it.

Reply:
> separate command

I think it's just because the compiletest bless functionality is newer and we just haven't migrated to it yet. But we also have an added feature where `cargo dev bless` only updates files that were updated after the last clippy build.